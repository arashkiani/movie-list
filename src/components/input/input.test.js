/*global shallow, mount, sinon */
import Component from '.'


describe('<Input />', () => {
	it('Match Snapshot', () => {
		const wrapper = shallow(<Component type="range" />)
		expect(wrapper).toMatchSnapshot()
	})
	it('Props pass', () => {
		const wrapper = mount(<Component type="checkbox" value={testString}/>)
    	expect(wrapper.find('input').props().value).toBe(testString)
   		wrapper.unmount();
	})
})