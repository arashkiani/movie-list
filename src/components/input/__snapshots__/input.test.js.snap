// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> Match Snapshot 1`] = `
<ForwardRef
  type="range"
/>
`;
